client/src/view/AvailableDrivers.js

```diff
@@ -59,23 +74,31 @@ export default () => (
 		</DivLeft>
 		
 		<Map>
-			Map goes here
+			Map
+		 <Iframe
+          key={apiKey}
+          title="directions"
+          frameborder="0"
+          src={destination === '' ?placeIframe : directionIframe}
+          allowfullscreen
+        /> 
 		</Map>
 	</WrapperTop>
 	
 	<AvailableDiv>
 		Available Drivers<br/>
-		<Avatar size={45} name="Niharika Nakka"/>
-		<Avatar size={45} name="Brett Lamy"/>
-		<Avatar size={45} name="Mohammad Suhail"/>
+			{["Niharika Nakka", "Brett Lamy", "Mohammad Suhail"].map(name =>
```

Comment:
In your storybook you pass this in properly but you don't use the `driver` property here.